lib/recurly/pricing/attach.js

```diff
@@ -47,9 +39,9 @@ function attach (el) {
     events.bind(elem, 'propertychange', change);
   });
 
-  this.detatch = detatch;
+  this.detach = detach;
 
-  change();
+  change(INIT_RUN);
```

Comment:
The gist of this bug is that this initial `change` run wasn't triggering any changes because of the IE event target finder fallback. It should target the `<script>` element that is running this code as the target element, thus never triggering the initial run detection in the `target` method. This makes the intent far clearer IMO.